fab/fabfile.py

```diff
@@ -184,6 +184,12 @@ def softlayer():
 @task
 def icds():
     """www.icds-cas.gov.in"""
+    # 05h30 - 20h30 IST
```

Comment:
did something similar for enikshay here https://github.com/dimagi/commcare-hq-deploy/blob/master/fab/fabfile.py#L245

Reply:
thanks, updated